### PR TITLE
codecrafters: add page

### PR DESCRIPTION
**Pull request by andy1li**
- [x] The page are in the correct platform directories: `common`
- [x] The page have at most 8 examples (3).
- [x] The page description have links to documentation or a homepage.
- [x] The page follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented:** v34-8f6a72f

### CODE REVIEW

**Review by spageektti — APPROVED** [review]
Hi. Thanks for your contribution.

File: pages/common/codecrafters.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# codecrafters
+
+> Practice writing complex software.
+> More information: <https://codecrafters.io/>.
```
Comment (spageektti) [inline pages/common/codecrafters.md:4]:
```suggestion
> More information: <https://docs.codecrafters.io/cli/usage>.
```

File: pages/common/codecrafters.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`codecrafters test`
+
+- Run tests for all previous stages and the current stage without committing changes:
+
+`codecrafters test --previous`
+
+- Commit changes and submit, to move to the next stage:
+
+`codecrafters submit`
```
Comment (spageektti) [inline pages/common/codecrafters.md:16]:
```suggestion
`codecrafters submit`

- Display help:
`codecrafters help`
```

Comment (andy1li) [reply pages/common/codecrafters.md:16]: Our cli is pretty simple. If people run help, they will get the same info as this tldr page, so not much value in list `help` here.

File: pages/common/codecrafters.md (hunk 1 above)
Comment (andy1li) [reply pages/common/codecrafters.md:4]: Our cli is pretty simple. If people go to the doc page, they will get the same info as this tldr page, so not much value in pointing to doc.

**Pull request merged**